_posts/2015-06-20-mental-number-line.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: Mass muddles mental number line
+author: neuroamanda
+---
+
+_Judgment of number influenced by other magnitudes and senses_
+
+People appear to have an inherent bias when it comes to number.
+They tend to count from left to right and are quicker to react in psychological tests when smaller numbers are on the left and larger numbers are on the right, at least in Western cultures; [chicks also display this same behavior](http://www.aaas.org/news/humans-chicks-count-left-right).
```

Comment:
the citation is the very first paragraph of the paper i am summarizing here... hmm.